src/lib/rnp.cpp

```diff
@@ -70,6 +70,13 @@
         RNP_LOG_FD(fp, __VA_ARGS__); \
     } while (0)
 
+#if defined(RNP_EXPERIMENTAL_CRYPTO_REFRESH) && !defined(ENABLE_CRYPTO_REFRESH)
```

Comment:
(This is a suggestion to consider)

IMHO it will be much easier to support a single flags
Something like this:

```
#include <iostream>

//   Defintion of CRYPTO_REFRESH options
#define ENABLE_CRYPTO_REFRESH 0
#define ENABLE_EXPERIMENTAL_CRYPTO_REFRESH 1
#define DISABLE_CRYPTO_REFRESH  2

//   Selection of CRYPTO_REFRESH option
#define RNP_CRYPTO_REFRESH RNP_CRYPTO_REFRESH


int main(int, char**)
{
#if RNP_CRYPTO_REFRESH == ENABLE_CRYPTO_REFRESH
    std::cout << "ENABLE_CRYPTO_REFRESH" << std::endl;
#elif RNP_CRYPTO_REFRESH == ENABLE_EXPERIMENTAL_CRYPTO_REFRESH
    std::cout << "ENABLE_EXPERIMENTAL_CRYPTO_REFRESH" << std::endl;
#else
    std::cout << "DISABLE_CRYPTO_REFRESH (" << RNP_CRYPTO_REFRESH << ")" << std::endl;
#endif
return 0;
}
```

Reply:
@maxirmx Thanks for noticing this, but it's suggested to work in a bit different way: both defines are for the same purpose, and this check is to make sure that they are in sync. The problem here is that one goes to the `config.h.in`, which is not installed and used only during compilation, and second is needed to be installed and used by another installed header `rnp.h`. This is needed to hide some function declarations from the `rnp.h`. So it's not the best solution, but I didn't find any better idea. Maybe you have some suggestion?

Reply:
If it is a check that both defines are in sync I guess it shall be 
```
#if defined(RNP_EXPERIMENTAL_CRYPTO_REFRESH) != defined(ENABLE_CRYPTO_REFRESH)
```

Reply:
@maxirmx Indeed, thanks! Re-pushed.